relnotes/delayed-suspender.migration.md

```diff
@@ -0,0 +1,18 @@
+## `DelayedSuspender` requires the request event dispatcher
+
+`swarm.neo.util.DelayedSuspender`
+
+Previously, using the `DelayedSuspender` with the request event dispatcher was
+buggy, as the suspender did not un/register the suspended fiber with the event
+dispatcher. This patch adapts the code to require the `DelayedSuspender` to be
+told about the existence of the request event dispatcher that's in use.
+
+Initialisation code like:
+```
+auto suspender = DelayedSuspender(fiber);
+```
+should be changed to:
+```
+auto suspender = DelayedSuspender(fiber, &request_event_dispatcher,
```

Comment:
`DelayedSuspender(&request_event_dispatcher, this.outer.conn, fiber, ...` (the order is now different, and it was missing `RoC.EventDispatcher` field).

Reply:
Updated.